src/libraries/System.IO.Compression.Brotli/src/System/IO/Compression/enc/BrotliEncoder.cs

```diff
@@ -93,6 +103,13 @@ internal void SetWindow(int window)
             }
         }
 
+        /// <summary>Gets the maximum expected compressed length for the provided input size.</summary>
+        /// <param name="inputSize">The input size to get the maximum expected compressed length from. Must be greater or equal than 0 and less or equal than <see cref="System.Int32.MaxValue" /> - 515.</param>
```

Comment:
The DocIDs use the full name of built-in types. I'll have to find and replace all full-named primitives to their alias, to avoid this error.

```suggestion
        /// <param name="inputSize">The input size to get the maximum expected compressed length from. Must be greater or equal than 0 and less or equal than <see cref="int.MaxValue" /> - 515.</param>
```